main/memcached/APKBUILD

```diff
@@ -16,13 +16,23 @@ source="https://www.memcached.org/files/${pkgname}-${pkgver}.tar.gz
 builddir="${srcdir}/${pkgname}-${pkgver}"
 
 build() {
+	# extstore is broken on 32-bit
+	# https://github.com/memcached/memcached/issues/319
+	local _extstore=''
+	case "$(uname -m)" in
```

Comment:
`uname` should not be used as it's "running system". Use instead `$CARCH`. See e.g. `main/x264` on how to use it. However, I am not sure if we want to keep this kind of feature in disparity between arches. Is there use case for this, or is this just 'let's enable things that exist'?